core/src/test/java/org/testcontainers/junit/DockerComposeContainerTest.java

```diff
@@ -33,4 +35,15 @@ public void testGetServicePort() {
         assertNotNull("Port is set for service with instance number", serviceWithoutInstancePort);
         assertEquals("Service ports are the same", serviceWithInstancePort, serviceWithoutInstancePort);
     }
+
+    @Test
+    public void testGetContainerByServiceName() {
+        String existingServiceName = "db_1";
+        assertTrue(format("Container should be found by service name %s", existingServiceName),
```

Comment:
could you please also assert some properties of the returned `ContainerState` (e.g. `getExposedPorts()`) to ensure that the returned instance is the correct one (and not Redis for example)

Reply:
Added additional check to compare if exported ports belong to correct container we are checking against